dGame/dComponents/RebuildComponent.cpp

```diff
@@ -22,6 +22,18 @@ RebuildComponent::RebuildComponent(Entity* entity) : Component(entity) {
 	{
 		m_Precondition = new PreconditionExpression(GeneralUtils::UTF16ToWTF8(checkPreconditions));
 	}
+
+	// Should a setting that has the build activator position exist, fetch that setting here and parse it for position.
+	// It is assumed that the user who sets this setting uses the correct character delimiter (character 31)
+	auto positionAsVector = GeneralUtils::SplitString(m_Parent->GetVarAsString(u"rebuild_activators"), 31);
+	if (positionAsVector.size() == 3) {
+		m_ActivatorPosition.x = std::stof(positionAsVector[0]);
+		m_ActivatorPosition.y = std::stof(positionAsVector[1]);
+		m_ActivatorPosition.z = std::stof(positionAsVector[2]);
```

Comment:
Can you add a try/catch around these? bad config values shouldn't cause exceptions

Reply:
Yeah sure.  Will toss those in so they default to using the quickbuilds spawn point should the position be bad

Reply:
Use GeneralUtils::TryParse